### DISCUSSION TITLE
Monitoring Worker Nodes

### DISCUSSION THREAD

**Discussion by mlopez-eb** [concourse/concourse#8505] [General]
Does anyone have a monitoring setup that works well for alerting of workers either down or stuck? There isn't much that I saw in the monitoring docs. I'm sure we could finagle something, but I was wondering if there was a known and efficient way.

Our Concourse CI is running out of AWS EKS, installed through Helm.

Please let me know if versioning and such is needed.

Thank you!

**Comment by ritik872000** [comment]
Hi, you can use AWS CloudWatch for monitoring and AWS SNS for alerting,

you need to use the concourse API for logging in to your concourse and some python code to extract the metrics and then you can create alarms based on that and integrate with SNS for alerting
